agents/trainer/task.py

```diff
@@ -0,0 +1,319 @@
+                    "The base directory in which to write logs and "
+                    "checkpoints.")
+flags.DEFINE_string("hparam_set_id", "pybullet_kuka_ff",
+                    "The name of the config object to be used to parameterize "
+                    "the run.")
+flags.DEFINE_string("run_base_tag",
+                    datetime.datetime.now().strftime('%Y%m%dT%H%M%S'),
+                    "Base tag to prepend to logs dir folder name. Defaults "
+                    "to timestamp.")
+flags.DEFINE_boolean("env_processes", True,
+                     "Step environments in separate processes to circumvent "
+                     "the GIL.")
+flags.DEFINE_boolean("sync_replicas", False,
+                     "Use the sync_replicas (synchronized replicas) mode, "
+                     "wherein the parameter updates from workers are "
+                     "aggregated before applied to avoid stale gradients.")
+flags.DEFINE_integer("num_gpus", 0,
+                     "Total number of gpus for each machine."
+                     "If you don't use GPU, please set it to '0'")
+flags.DEFINE_integer("save_checkpoint_secs", 600,
+                     "Number of seconds between checkpoint save.")
+flags.DEFINE_boolean("use_monitored_training_session", True,
+                     "Whether to use tf.train.MonitoredTrainingSession to "
+                     "manage the training session. If not, use "
+                     "tf.train.Supervisor.")
+
+flags.DEFINE_boolean("log_device_placement", False,
+                     "Whether to output logs listing the devices on which "
+                     "variables are placed.")
+flags.DEFINE_boolean("debug", True,
+                     "Run in debug mode.")
+flags.DEFINE_boolean("dump_dependency_versions", False,
+                     "Run in debug mode.")
+
+# Render
+flags.DEFINE_integer("render_secs", 600,
+                     "Number of seconds between triggering render jobs.")
+flags.DEFINE_string("render_out_dir", None,
+                    "The path to which to copy generated renders.")
+
+# Algorithm
+flags.DEFINE_string("algorithm", "agents.ppo.PPOAlgorithm",
+                    "The name of the algorithm to use.")
+flags.DEFINE_integer("num_agents", 30,
+                     "The number of agents to use.")
+flags.DEFINE_integer("eval_episodes", 25,
+                     "The number of eval episodes to use.")
+flags.DEFINE_string("env", "AntBulletEnv-v0",
+                    "The gym / bullet simulation environment to use.")
+flags.DEFINE_integer("max_length", 1000,
+                     "The maximum length of an episode.")
+flags.DEFINE_integer("steps", 10000000,
+                     "The number of steps.")
+
+# Network
+flags.DEFINE_string("network", "agents.scripts.networks.feed_forward_gaussian",
+                    "The registered network name to use for policy and value.")
+# TODO: Make it possible to pass policy and value layers via hparam config.
```

Comment:
If this is not supported yet, it should probably be removed from the PR.

Reply:
Sounds good.